test/readme.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-env node, mocha */
+
+var expect = require('chai').expect;
+var Exonum = require('..');
+
+describe('Examples from README.md', function() {
```

Comment:
Nitpick: some linters mark as warning fragments that are delimited by empty lines from both sides. Propose to format as follows:

```javascript
describe('Examples from README.md', function() {
    describe('Custom type section', function() {
        var User = Exonum.newType({
        // other lines
    });

    describe('Transaction section', function() {
        // ...
    });
});
```

Reply:
done, thanks